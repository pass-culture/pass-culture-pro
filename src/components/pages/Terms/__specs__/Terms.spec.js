import React from 'react'
import { shallow } from 'enzyme'
import Terms from '../Terms'

describe('src | components | pages | Terms', () => {
  it('should match snapshot', () => {
    // when
    const wrapper = shallow(<Terms />)

    // then
    expect(wrapper).toMatchSnapshot()
  })
})
