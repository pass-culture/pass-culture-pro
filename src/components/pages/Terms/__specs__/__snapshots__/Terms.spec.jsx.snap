// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src | components | pages | Terms should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Terms
    lastDeployedCommit="##LAST_DEPLOYED_COMMIT##"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "backButton": true,
      "children": Array [
        <header>
          <h1>
            Mentions légales
          </h1>
        </header>,
        <div
          className="content"
        >
          <ReactMarkdown
            astPlugins={Array []}
            escapeHtml={true}
            parserOptions={Object {}}
            plugins={Array []}
            rawSourcePos={false}
            renderers={Object {}}
            skipHtml={false}
            source="
# Titre section

## Titre Sous-section

\\"Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...\\"
\\"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain...\\"
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eget hendrerit urna, ut volutpat ligula. Aenean
egestas in erat vitae fringilla. Donec vel ante diam. Integer sit amet massa vitae sem pretium faucibus nec quis
nibh. Sed ut turpis quis odio vestibulum pulvinar. Mauris id urna turpis. Vivamus lobortis sit amet eros ut auctor.
Praesent sapien nisl, scelerisque id risus at, luctus fringilla lorem.

## Titre Sous-section

Donec fermentum at est convallis sollicitudin. Nulla maximus leo eget erat malesuada, in cursus dolor mollis.
Duis eu arcu ac leo condimentum pellentesque. In dolor neque, molestie ac ultrices ut, eleifend sit amet sapien.
Etiam congue felis nec metus fringilla mollis. Sed id sagittis diam. Vestibulum maximus nisi vel ante tempor,
eget rhoncus risus iaculis. Phasellus dolor lectus, convallis at suscipit at, viverra non ipsum. Nullam congue
laoreet lacus, eu dapibus leo.

Aenean lobortis sit amet sapien vel lobortis. Integer sagittis, sapien id accumsan facilisis, erat orci tristique
justo, a ultrices velit sem et enim. Suspendisse ut cursus nulla, id pulvinar mi. Curabitur fermentum id eros non
sollicitudin. Nullam placerat placerat urna a tempus. Cras vitae tortor erat. Nam sed ex et justo luctus consectetur
eget in massa. Phasellus nec erat tempus, luctus nisi non, placerat sapien. Vestibulum ultricies ullamcorper enim,
eget cursus nulla maximus ut. Etiam rhoncus eleifend feugiat. Curabitur ultricies ante quam, non fringilla nisl aliquam
non. Proin pharetra consectetur auctor. Proin finibus elementum sollicitudin. Maecenas ac eleifend tortor. Pellentesque
efficitur turpis id lorem maximus commodo sed vitae ipsum. Proin sed enim sapien.

Nullam molestie, enim eget consequat elementum, mi neque dictum mi, id efficitur magna lectus nec elit.
Cras quis eleifend magna. Duis iaculis lorem at tristique condimentum. Aliquam interdum nisi at pharetra
venenatis. Curabitur elementum at felis ut porttitor. Quisque volutpat tempus orci, eu egestas sem consequat
iaculis. Fusce ac convallis quam, sed consectetur sem. Aenean suscipit vel leo quis convallis. Cras posuere
lorem at nisl auctor luctus sit amet in metus. Donec eu efficitur magna, at iaculis purus. Donec nunc erat,
maximus vitae nibh ac, ultrices pharetra lorem. Vestibulum vel consectetur nulla, ac commodo arcu. Aenean vel
magna viverra, rhoncus nisi quis, ornare nisi. In rhoncus urna metus. Fusce sed posuere orci.

Donec interdum sodales ligula, faucibus imperdiet leo mattis in. Cras rhoncus, tortor vitae porta ultrices,
ex nisl imperdiet est, vitae scelerisque metus elit sit amet urna. Cras posuere pulvinar risus id sagittis.
Ut lacinia dapibus tempor. Maecenas egestas sapien non diam porta, sed porttitor orci aliquet. Fusce quis
sagittis massa. Duis eleifend iaculis urna eu lacinia. Phasellus bibendum felis in eros fermentum rhoncus.
Nam et consectetur nisl. Proin eget gravida ex. Pellentesque a leo enim. Aliquam sit amet felis arcu. Nunc
feugiat orci ut diam ullamcorper consequat.
"
            sourcePos={false}
            transformLinkUri={[Function]}
          />
          <div
            className="mt16"
          >
            <p
              className="text-right"
            >
              pass Culture - ##LAST_DEPLOYED_COMMIT##
            </p>
          </div>
        </div>,
      ],
      "name": "terms",
      "noHeader": true,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1>
            Mentions légales
          </h1>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Mentions légales",
          },
          "ref": null,
          "rendered": "Mentions légales",
          "type": "h1",
        },
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <ReactMarkdown
              astPlugins={Array []}
              escapeHtml={true}
              parserOptions={Object {}}
              plugins={Array []}
              rawSourcePos={false}
              renderers={Object {}}
              skipHtml={false}
              source="
# Titre section

## Titre Sous-section

\\"Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...\\"
\\"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain...\\"
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eget hendrerit urna, ut volutpat ligula. Aenean
egestas in erat vitae fringilla. Donec vel ante diam. Integer sit amet massa vitae sem pretium faucibus nec quis
nibh. Sed ut turpis quis odio vestibulum pulvinar. Mauris id urna turpis. Vivamus lobortis sit amet eros ut auctor.
Praesent sapien nisl, scelerisque id risus at, luctus fringilla lorem.

## Titre Sous-section

Donec fermentum at est convallis sollicitudin. Nulla maximus leo eget erat malesuada, in cursus dolor mollis.
Duis eu arcu ac leo condimentum pellentesque. In dolor neque, molestie ac ultrices ut, eleifend sit amet sapien.
Etiam congue felis nec metus fringilla mollis. Sed id sagittis diam. Vestibulum maximus nisi vel ante tempor,
eget rhoncus risus iaculis. Phasellus dolor lectus, convallis at suscipit at, viverra non ipsum. Nullam congue
laoreet lacus, eu dapibus leo.

Aenean lobortis sit amet sapien vel lobortis. Integer sagittis, sapien id accumsan facilisis, erat orci tristique
justo, a ultrices velit sem et enim. Suspendisse ut cursus nulla, id pulvinar mi. Curabitur fermentum id eros non
sollicitudin. Nullam placerat placerat urna a tempus. Cras vitae tortor erat. Nam sed ex et justo luctus consectetur
eget in massa. Phasellus nec erat tempus, luctus nisi non, placerat sapien. Vestibulum ultricies ullamcorper enim,
eget cursus nulla maximus ut. Etiam rhoncus eleifend feugiat. Curabitur ultricies ante quam, non fringilla nisl aliquam
non. Proin pharetra consectetur auctor. Proin finibus elementum sollicitudin. Maecenas ac eleifend tortor. Pellentesque
efficitur turpis id lorem maximus commodo sed vitae ipsum. Proin sed enim sapien.

Nullam molestie, enim eget consequat elementum, mi neque dictum mi, id efficitur magna lectus nec elit.
Cras quis eleifend magna. Duis iaculis lorem at tristique condimentum. Aliquam interdum nisi at pharetra
venenatis. Curabitur elementum at felis ut porttitor. Quisque volutpat tempus orci, eu egestas sem consequat
iaculis. Fusce ac convallis quam, sed consectetur sem. Aenean suscipit vel leo quis convallis. Cras posuere
lorem at nisl auctor luctus sit amet in metus. Donec eu efficitur magna, at iaculis purus. Donec nunc erat,
maximus vitae nibh ac, ultrices pharetra lorem. Vestibulum vel consectetur nulla, ac commodo arcu. Aenean vel
magna viverra, rhoncus nisi quis, ornare nisi. In rhoncus urna metus. Fusce sed posuere orci.

Donec interdum sodales ligula, faucibus imperdiet leo mattis in. Cras rhoncus, tortor vitae porta ultrices,
ex nisl imperdiet est, vitae scelerisque metus elit sit amet urna. Cras posuere pulvinar risus id sagittis.
Ut lacinia dapibus tempor. Maecenas egestas sapien non diam porta, sed porttitor orci aliquet. Fusce quis
sagittis massa. Duis eleifend iaculis urna eu lacinia. Phasellus bibendum felis in eros fermentum rhoncus.
Nam et consectetur nisl. Proin eget gravida ex. Pellentesque a leo enim. Aliquam sit amet felis arcu. Nunc
feugiat orci ut diam ullamcorper consequat.
"
              sourcePos={false}
              transformLinkUri={[Function]}
            />,
            <div
              className="mt16"
            >
              <p
                className="text-right"
              >
                pass Culture - ##LAST_DEPLOYED_COMMIT##
              </p>
            </div>,
          ],
          "className": "content",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "astPlugins": Array [],
              "escapeHtml": true,
              "parserOptions": Object {},
              "plugins": Array [],
              "rawSourcePos": false,
              "renderers": Object {},
              "skipHtml": false,
              "source": "
# Titre section

## Titre Sous-section

\\"Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...\\"
\\"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain...\\"
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eget hendrerit urna, ut volutpat ligula. Aenean
egestas in erat vitae fringilla. Donec vel ante diam. Integer sit amet massa vitae sem pretium faucibus nec quis
nibh. Sed ut turpis quis odio vestibulum pulvinar. Mauris id urna turpis. Vivamus lobortis sit amet eros ut auctor.
Praesent sapien nisl, scelerisque id risus at, luctus fringilla lorem.

## Titre Sous-section

Donec fermentum at est convallis sollicitudin. Nulla maximus leo eget erat malesuada, in cursus dolor mollis.
Duis eu arcu ac leo condimentum pellentesque. In dolor neque, molestie ac ultrices ut, eleifend sit amet sapien.
Etiam congue felis nec metus fringilla mollis. Sed id sagittis diam. Vestibulum maximus nisi vel ante tempor,
eget rhoncus risus iaculis. Phasellus dolor lectus, convallis at suscipit at, viverra non ipsum. Nullam congue
laoreet lacus, eu dapibus leo.

Aenean lobortis sit amet sapien vel lobortis. Integer sagittis, sapien id accumsan facilisis, erat orci tristique
justo, a ultrices velit sem et enim. Suspendisse ut cursus nulla, id pulvinar mi. Curabitur fermentum id eros non
sollicitudin. Nullam placerat placerat urna a tempus. Cras vitae tortor erat. Nam sed ex et justo luctus consectetur
eget in massa. Phasellus nec erat tempus, luctus nisi non, placerat sapien. Vestibulum ultricies ullamcorper enim,
eget cursus nulla maximus ut. Etiam rhoncus eleifend feugiat. Curabitur ultricies ante quam, non fringilla nisl aliquam
non. Proin pharetra consectetur auctor. Proin finibus elementum sollicitudin. Maecenas ac eleifend tortor. Pellentesque
efficitur turpis id lorem maximus commodo sed vitae ipsum. Proin sed enim sapien.

Nullam molestie, enim eget consequat elementum, mi neque dictum mi, id efficitur magna lectus nec elit.
Cras quis eleifend magna. Duis iaculis lorem at tristique condimentum. Aliquam interdum nisi at pharetra
venenatis. Curabitur elementum at felis ut porttitor. Quisque volutpat tempus orci, eu egestas sem consequat
iaculis. Fusce ac convallis quam, sed consectetur sem. Aenean suscipit vel leo quis convallis. Cras posuere
lorem at nisl auctor luctus sit amet in metus. Donec eu efficitur magna, at iaculis purus. Donec nunc erat,
maximus vitae nibh ac, ultrices pharetra lorem. Vestibulum vel consectetur nulla, ac commodo arcu. Aenean vel
magna viverra, rhoncus nisi quis, ornare nisi. In rhoncus urna metus. Fusce sed posuere orci.

Donec interdum sodales ligula, faucibus imperdiet leo mattis in. Cras rhoncus, tortor vitae porta ultrices,
ex nisl imperdiet est, vitae scelerisque metus elit sit amet urna. Cras posuere pulvinar risus id sagittis.
Ut lacinia dapibus tempor. Maecenas egestas sapien non diam porta, sed porttitor orci aliquet. Fusce quis
sagittis massa. Duis eleifend iaculis urna eu lacinia. Phasellus bibendum felis in eros fermentum rhoncus.
Nam et consectetur nisl. Proin eget gravida ex. Pellentesque a leo enim. Aliquam sit amet felis arcu. Nunc
feugiat orci ut diam ullamcorper consequat.
",
              "sourcePos": false,
              "transformLinkUri": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <p
                className="text-right"
              >
                pass Culture - ##LAST_DEPLOYED_COMMIT##
              </p>,
              "className": "mt16",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "pass Culture - ##LAST_DEPLOYED_COMMIT##",
                "className": "text-right",
              },
              "ref": null,
              "rendered": "pass Culture - ##LAST_DEPLOYED_COMMIT##",
              "type": "p",
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "backButton": true,
        "children": Array [
          <header>
            <h1>
              Mentions légales
            </h1>
          </header>,
          <div
            className="content"
          >
            <ReactMarkdown
              astPlugins={Array []}
              escapeHtml={true}
              parserOptions={Object {}}
              plugins={Array []}
              rawSourcePos={false}
              renderers={Object {}}
              skipHtml={false}
              source="
# Titre section

## Titre Sous-section

\\"Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...\\"
\\"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain...\\"
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eget hendrerit urna, ut volutpat ligula. Aenean
egestas in erat vitae fringilla. Donec vel ante diam. Integer sit amet massa vitae sem pretium faucibus nec quis
nibh. Sed ut turpis quis odio vestibulum pulvinar. Mauris id urna turpis. Vivamus lobortis sit amet eros ut auctor.
Praesent sapien nisl, scelerisque id risus at, luctus fringilla lorem.

## Titre Sous-section

Donec fermentum at est convallis sollicitudin. Nulla maximus leo eget erat malesuada, in cursus dolor mollis.
Duis eu arcu ac leo condimentum pellentesque. In dolor neque, molestie ac ultrices ut, eleifend sit amet sapien.
Etiam congue felis nec metus fringilla mollis. Sed id sagittis diam. Vestibulum maximus nisi vel ante tempor,
eget rhoncus risus iaculis. Phasellus dolor lectus, convallis at suscipit at, viverra non ipsum. Nullam congue
laoreet lacus, eu dapibus leo.

Aenean lobortis sit amet sapien vel lobortis. Integer sagittis, sapien id accumsan facilisis, erat orci tristique
justo, a ultrices velit sem et enim. Suspendisse ut cursus nulla, id pulvinar mi. Curabitur fermentum id eros non
sollicitudin. Nullam placerat placerat urna a tempus. Cras vitae tortor erat. Nam sed ex et justo luctus consectetur
eget in massa. Phasellus nec erat tempus, luctus nisi non, placerat sapien. Vestibulum ultricies ullamcorper enim,
eget cursus nulla maximus ut. Etiam rhoncus eleifend feugiat. Curabitur ultricies ante quam, non fringilla nisl aliquam
non. Proin pharetra consectetur auctor. Proin finibus elementum sollicitudin. Maecenas ac eleifend tortor. Pellentesque
efficitur turpis id lorem maximus commodo sed vitae ipsum. Proin sed enim sapien.

Nullam molestie, enim eget consequat elementum, mi neque dictum mi, id efficitur magna lectus nec elit.
Cras quis eleifend magna. Duis iaculis lorem at tristique condimentum. Aliquam interdum nisi at pharetra
venenatis. Curabitur elementum at felis ut porttitor. Quisque volutpat tempus orci, eu egestas sem consequat
iaculis. Fusce ac convallis quam, sed consectetur sem. Aenean suscipit vel leo quis convallis. Cras posuere
lorem at nisl auctor luctus sit amet in metus. Donec eu efficitur magna, at iaculis purus. Donec nunc erat,
maximus vitae nibh ac, ultrices pharetra lorem. Vestibulum vel consectetur nulla, ac commodo arcu. Aenean vel
magna viverra, rhoncus nisi quis, ornare nisi. In rhoncus urna metus. Fusce sed posuere orci.

Donec interdum sodales ligula, faucibus imperdiet leo mattis in. Cras rhoncus, tortor vitae porta ultrices,
ex nisl imperdiet est, vitae scelerisque metus elit sit amet urna. Cras posuere pulvinar risus id sagittis.
Ut lacinia dapibus tempor. Maecenas egestas sapien non diam porta, sed porttitor orci aliquet. Fusce quis
sagittis massa. Duis eleifend iaculis urna eu lacinia. Phasellus bibendum felis in eros fermentum rhoncus.
Nam et consectetur nisl. Proin eget gravida ex. Pellentesque a leo enim. Aliquam sit amet felis arcu. Nunc
feugiat orci ut diam ullamcorper consequat.
"
              sourcePos={false}
              transformLinkUri={[Function]}
            />
            <div
              className="mt16"
            >
              <p
                className="text-right"
              >
                pass Culture - ##LAST_DEPLOYED_COMMIT##
              </p>
            </div>
          </div>,
        ],
        "name": "terms",
        "noHeader": true,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1>
              Mentions légales
            </h1>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Mentions légales",
            },
            "ref": null,
            "rendered": "Mentions légales",
            "type": "h1",
          },
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <ReactMarkdown
                astPlugins={Array []}
                escapeHtml={true}
                parserOptions={Object {}}
                plugins={Array []}
                rawSourcePos={false}
                renderers={Object {}}
                skipHtml={false}
                source="
# Titre section

## Titre Sous-section

\\"Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...\\"
\\"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain...\\"
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eget hendrerit urna, ut volutpat ligula. Aenean
egestas in erat vitae fringilla. Donec vel ante diam. Integer sit amet massa vitae sem pretium faucibus nec quis
nibh. Sed ut turpis quis odio vestibulum pulvinar. Mauris id urna turpis. Vivamus lobortis sit amet eros ut auctor.
Praesent sapien nisl, scelerisque id risus at, luctus fringilla lorem.

## Titre Sous-section

Donec fermentum at est convallis sollicitudin. Nulla maximus leo eget erat malesuada, in cursus dolor mollis.
Duis eu arcu ac leo condimentum pellentesque. In dolor neque, molestie ac ultrices ut, eleifend sit amet sapien.
Etiam congue felis nec metus fringilla mollis. Sed id sagittis diam. Vestibulum maximus nisi vel ante tempor,
eget rhoncus risus iaculis. Phasellus dolor lectus, convallis at suscipit at, viverra non ipsum. Nullam congue
laoreet lacus, eu dapibus leo.

Aenean lobortis sit amet sapien vel lobortis. Integer sagittis, sapien id accumsan facilisis, erat orci tristique
justo, a ultrices velit sem et enim. Suspendisse ut cursus nulla, id pulvinar mi. Curabitur fermentum id eros non
sollicitudin. Nullam placerat placerat urna a tempus. Cras vitae tortor erat. Nam sed ex et justo luctus consectetur
eget in massa. Phasellus nec erat tempus, luctus nisi non, placerat sapien. Vestibulum ultricies ullamcorper enim,
eget cursus nulla maximus ut. Etiam rhoncus eleifend feugiat. Curabitur ultricies ante quam, non fringilla nisl aliquam
non. Proin pharetra consectetur auctor. Proin finibus elementum sollicitudin. Maecenas ac eleifend tortor. Pellentesque
efficitur turpis id lorem maximus commodo sed vitae ipsum. Proin sed enim sapien.

Nullam molestie, enim eget consequat elementum, mi neque dictum mi, id efficitur magna lectus nec elit.
Cras quis eleifend magna. Duis iaculis lorem at tristique condimentum. Aliquam interdum nisi at pharetra
venenatis. Curabitur elementum at felis ut porttitor. Quisque volutpat tempus orci, eu egestas sem consequat
iaculis. Fusce ac convallis quam, sed consectetur sem. Aenean suscipit vel leo quis convallis. Cras posuere
lorem at nisl auctor luctus sit amet in metus. Donec eu efficitur magna, at iaculis purus. Donec nunc erat,
maximus vitae nibh ac, ultrices pharetra lorem. Vestibulum vel consectetur nulla, ac commodo arcu. Aenean vel
magna viverra, rhoncus nisi quis, ornare nisi. In rhoncus urna metus. Fusce sed posuere orci.

Donec interdum sodales ligula, faucibus imperdiet leo mattis in. Cras rhoncus, tortor vitae porta ultrices,
ex nisl imperdiet est, vitae scelerisque metus elit sit amet urna. Cras posuere pulvinar risus id sagittis.
Ut lacinia dapibus tempor. Maecenas egestas sapien non diam porta, sed porttitor orci aliquet. Fusce quis
sagittis massa. Duis eleifend iaculis urna eu lacinia. Phasellus bibendum felis in eros fermentum rhoncus.
Nam et consectetur nisl. Proin eget gravida ex. Pellentesque a leo enim. Aliquam sit amet felis arcu. Nunc
feugiat orci ut diam ullamcorper consequat.
"
                sourcePos={false}
                transformLinkUri={[Function]}
              />,
              <div
                className="mt16"
              >
                <p
                  className="text-right"
                >
                  pass Culture - ##LAST_DEPLOYED_COMMIT##
                </p>
              </div>,
            ],
            "className": "content",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "astPlugins": Array [],
                "escapeHtml": true,
                "parserOptions": Object {},
                "plugins": Array [],
                "rawSourcePos": false,
                "renderers": Object {},
                "skipHtml": false,
                "source": "
# Titre section

## Titre Sous-section

\\"Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...\\"
\\"There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain...\\"
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eget hendrerit urna, ut volutpat ligula. Aenean
egestas in erat vitae fringilla. Donec vel ante diam. Integer sit amet massa vitae sem pretium faucibus nec quis
nibh. Sed ut turpis quis odio vestibulum pulvinar. Mauris id urna turpis. Vivamus lobortis sit amet eros ut auctor.
Praesent sapien nisl, scelerisque id risus at, luctus fringilla lorem.

## Titre Sous-section

Donec fermentum at est convallis sollicitudin. Nulla maximus leo eget erat malesuada, in cursus dolor mollis.
Duis eu arcu ac leo condimentum pellentesque. In dolor neque, molestie ac ultrices ut, eleifend sit amet sapien.
Etiam congue felis nec metus fringilla mollis. Sed id sagittis diam. Vestibulum maximus nisi vel ante tempor,
eget rhoncus risus iaculis. Phasellus dolor lectus, convallis at suscipit at, viverra non ipsum. Nullam congue
laoreet lacus, eu dapibus leo.

Aenean lobortis sit amet sapien vel lobortis. Integer sagittis, sapien id accumsan facilisis, erat orci tristique
justo, a ultrices velit sem et enim. Suspendisse ut cursus nulla, id pulvinar mi. Curabitur fermentum id eros non
sollicitudin. Nullam placerat placerat urna a tempus. Cras vitae tortor erat. Nam sed ex et justo luctus consectetur
eget in massa. Phasellus nec erat tempus, luctus nisi non, placerat sapien. Vestibulum ultricies ullamcorper enim,
eget cursus nulla maximus ut. Etiam rhoncus eleifend feugiat. Curabitur ultricies ante quam, non fringilla nisl aliquam
non. Proin pharetra consectetur auctor. Proin finibus elementum sollicitudin. Maecenas ac eleifend tortor. Pellentesque
efficitur turpis id lorem maximus commodo sed vitae ipsum. Proin sed enim sapien.

Nullam molestie, enim eget consequat elementum, mi neque dictum mi, id efficitur magna lectus nec elit.
Cras quis eleifend magna. Duis iaculis lorem at tristique condimentum. Aliquam interdum nisi at pharetra
venenatis. Curabitur elementum at felis ut porttitor. Quisque volutpat tempus orci, eu egestas sem consequat
iaculis. Fusce ac convallis quam, sed consectetur sem. Aenean suscipit vel leo quis convallis. Cras posuere
lorem at nisl auctor luctus sit amet in metus. Donec eu efficitur magna, at iaculis purus. Donec nunc erat,
maximus vitae nibh ac, ultrices pharetra lorem. Vestibulum vel consectetur nulla, ac commodo arcu. Aenean vel
magna viverra, rhoncus nisi quis, ornare nisi. In rhoncus urna metus. Fusce sed posuere orci.

Donec interdum sodales ligula, faucibus imperdiet leo mattis in. Cras rhoncus, tortor vitae porta ultrices,
ex nisl imperdiet est, vitae scelerisque metus elit sit amet urna. Cras posuere pulvinar risus id sagittis.
Ut lacinia dapibus tempor. Maecenas egestas sapien non diam porta, sed porttitor orci aliquet. Fusce quis
sagittis massa. Duis eleifend iaculis urna eu lacinia. Phasellus bibendum felis in eros fermentum rhoncus.
Nam et consectetur nisl. Proin eget gravida ex. Pellentesque a leo enim. Aliquam sit amet felis arcu. Nunc
feugiat orci ut diam ullamcorper consequat.
",
                "sourcePos": false,
                "transformLinkUri": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p
                  className="text-right"
                >
                  pass Culture - ##LAST_DEPLOYED_COMMIT##
                </p>,
                "className": "mt16",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "pass Culture - ##LAST_DEPLOYED_COMMIT##",
                  "className": "text-right",
                },
                "ref": null,
                "rendered": "pass Culture - ##LAST_DEPLOYED_COMMIT##",
                "type": "p",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
